sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -99,6 +99,8 @@ object functions {
    * The passed in object is returned directly if it is already a [[Column]].
    * If the object is a Scala Symbol, it is converted into a [[Column]] also.
    * Otherwise, a new [[Column]] is created to represent the literal value.
+   * Different from [[lit]], this functions can handle all the types in Scala,
```

Comment:
`This different between this function and [[lit]] is that this function can handle parameterized scala types e.g.: List, Seq and Map.`